.github/workflows/main.yml

```diff
@@ -27,10 +23,10 @@ jobs:
         uses: actions/setup-java@v1
         with:
           java-version: 11
+      - name: Run Kotlin Linter
+        run: ./gradlew app:ktlintCheck --info
       - name: Generate debug build
         run: ./gradlew assembleDebug
-      - name: Run tests
-        run: ./gradlew test

```

Comment:
**question**: Does that mean we don't have tests anymore? Do we plan to add some later on?

Reply:
I have remove tests for now, because we have not tests about app (we have just one test to check if 2+2 =4)
@J-Jamet have added first test to test function to get permissions.
But this test have need to use emulator. I think is not necessary to start tests with emulator foreach commits.
If you want, i can open new discussion to talk to create new workflow in the futur only for tests (tests with and without emulator).